New activities functionality (#34)
daytistics/daytistics: README.md

You can find a full list of our features in the [documentation](http://docs.daytistics.com/)

> ## Project setting up

Reviewer: Please move the installation instructions in INSTALL.md under the daytistics core section

Reviewer: For example:

```markdown
# Installation and Deployment

## Prerequisites

[...]

## Development Installation
### Daytistics-Core
<-- here -->

To be continued...
```

You can assume that the repo has already been cloned.

> - Clone this repository

Reviewer: ```suggestion
```

> - Ensure Poetry is Properly Installed [Poetry Official Documentation](https://python-poetry.org/docs/#installing-with-the-official-installer)

Reviewer: ```suggestion
```

> - Navigate to the Project Directory: `cd my-project`

Reviewer: ```suggestion
- Navigate to the Project Directory: `cd daytistics/daytistics-core`
```

> - If you have a pyproject.toml file with dependencies listed, you can install them by running: `poetry install`

Reviewer: ```suggestion
- Install the dependencies by running `poetry install`
```
